src/com/lftechnology/java_training/jan5/ContentInArraysComparator.java

```diff
@@ -0,0 +1,113 @@
+package com.lftechnology.java_training.jan5;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Compare if two arrays have the same content (doesn't matter if they aren't in
+ * order)
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class ContentInArraysComparator {
+
+	public static void main(String[] args) {
+		ArrayStore arrayStore1 = new ArrayStore();
+		arrayStore1.setValueForArray("Enter value for array1");
+
+		ArrayStore arrayStore2 = new ArrayStore();
+		arrayStore2.setValueForArray("Enter value for array2");
+
+		arrayStore1.compareArrayValues(arrayStore2);
+	}
+
+}
+
+/**
+ * this class stores value in array and compares two array if they both have
+ * same content or not despite of random order
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+class ArrayStore {
+	private String[] arrayOfValues = new String[10];
+	private final Logger LOGGER = Logger.getLogger(ArrayStore.class.getName());
```

Comment:
Good Job with the LOGGER declaration , but you should add `static` to it so that only one instance is created per class. `private static final Logger LOGGER`